git236.spec

```diff
@@ -1075,6 +1075,10 @@ rmdir --ignore-fail-on-non-empty "$testdir"
 %{?with_docs:%{_pkgdocdir}/git-svn.html}
 
 %changelog
+* Tue Jan 20 2023 Danilo Braband <dbraband@gmail.com> - 2.36.4-1
```

Comment:
Can you fix this date please? Either the day is off or the date is off.

Reply:
Fixed it. Sorry.